### PR TITLE
[image] Move imageutils from sm to common

### PR DESCRIPTION
**Pull request by Eabramov84**
Signed-off-by: Yevgen Abramov <Yevgen_Abramov@epam.com>

### CODE REVIEW

**Review by MykolaSolyanko — APPROVED** [review]

File: image/image.go [hunk 1]
```diff
@@ -269,3 +336,21 @@ func getItemPath(destination, name string) (itemPath string, err error) {
 
 	return itemPath, nil
 }
+
+func unTarFromFile(tarArchive string, destination string) (err error) {
```
Comment (al1img) [inline image/image.go:340]: Could you move content to UnpackTarImage? I don't see any reason for this private function.

Comment (Eabramov84) [reply image/image.go:340]: done

File: image/image.go [hunk 2]
```diff
@@ -222,6 +230,79 @@ func UntarGZArchive(ctx context.Context, source, destination string) (err error)
 	}
 }
 
+// UnpackTarImage extract tar image.
```
Comment (al1img) [inline image/image.go:233]: extracts

Comment (Eabramov84) [reply image/image.go]: fixed

**Pull request merged**